components/organisms/o-payment.vue

```diff
@@ -220,6 +221,13 @@ export default {
     SfHeading,
     SfCheckbox
   },
+  computed: {
+    isUserExist () {
```

Comment:
We can modify the name to "currentUser", it will be more consistent.
m-modal-review.vue and m-update-password.vue both have same function with "currentUser" name for the similar check.

Reply:
Updated, please check.

Reply:
looks great !